Exception: STATUS_ACCESS_VIOLATION at rip=001800D23FB
rax=0000000600072520 rbx=000006FFFFE66878 rcx=000000001FFE4B81
rdx=00000000FFFFC480 rsi=000006FFFFF3FFFC rdi=0000000600148D98
r8 =FFFFF90600208D9C r9 =00000001803034C8 r10=0000000100000000
r11=000000041C361764 r12=000006FFFFE0E1F0 r13=00000003F224AD00
r14=000006FFFFE0E1E0 r15=000006FFFFF24A34
rbp=00000000FFFFC440 rsp=00000000FFFFC408
program=C:\cygwin64\bin\python2.7.exe, pid 35636, thread main
cs=0033 ds=002B es=002B fs=0053 gs=002B ss=002B
Stack trace:
Frame        Function    Args
000FFFFC440  001800D23FB (6FFFFE41510, 6FFFFF24A2E, 6FFFFE66878, 6FFFFE0E050)
000FFFFC440  0041C3617B4 (000FFFFC490, 000FFFFC480, 003F21E9AA5, 6FFFFE6F840)
000FFFFC4A0  0041C3618EC (000FFFFC4D8, 000FFFFC4E0, 003F2166204, 6FFFFE703D0)
000FFFFC4F0  0041C361B10 (6FFFFE41510, 6FFFFE8EBA8, 6FFFFFE9DA0, 6FFFFFE9DA8)
0060003A1B0  003F21D787A (6FFFFF2AED0, 6FFFFF15168, 6FFFFF15168, 003F224B4D0)
6FFFFE0E050  003F21D7AFC (003F214CF50, 01A0000000A, 00000000033, 00000000000)
00000000000  003F21D84E8 (00000000000, 6FFFFE4E050, 006000741C0, 00000000000)
6FFFFF15168  003F21D863F (001801C9FE3, 000FFFFCC5E, 000FFFFC96C, 000FFFFC97C)
6FFFFF15168  003F21FFA51 (0018032DAE0, 0060006DB80, 00080127BFB, 6FFFFF15168)
000FFFFCC5E  003F220147F (00000000000, 00180046750, 000FFFFCB00, 001801C1500)
6FFFFF160D0  003F2218A00 (00000000020, 30001000000FF00, 0018004843C, 000FFFFD680)
000FFFFCCC0  001800484AD (00000000000, 00000000000, 00000000000, 00000000000)
00000000000  0018004618C (00000000000, 00000000000, 00000000000, 00000000000)
000FFFFFFF0  00180046224 (00000000000, 00000000000, 00000000000, 00000000000)
End of stack trace
